flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/source/AbstractHBaseDynamicTableSource.java

```diff
@@ -114,4 +117,28 @@ public String asSummaryString() {
     public HBaseTableSchema getHBaseTableSchema() {
         return this.hbaseSchema;
     }
+
+    // get HBase table properties which start with prefix
+    public static Properties getHBaseClientProperties(
+            Map<String, String> tableOptions, String prefix) {
```

Comment:
I think we don't need the `prefix` parameter, the prefix should always be `properties.`.

Reply:
yes,i will modify